Tools/CMake/AMReXParallelBackends.cmake

```diff
@@ -171,12 +171,14 @@ endif ()
 #
 if (AMReX_HIP)
 
-   set(_valid_hip_compilers hipcc nvcc)
+   set(_valid_hip_compilers clang++ hipcc nvcc CC)
```

Comment:
`clang++ -x hip` defaults to C++11: https://github.com/ROCm-Developer-Tools/HIP/issues/2278

Need to work-around by either adding to the `COMPILE_OPTIONS` property or fixing `CXX_STANDARD` to C++14 (somewhat different to the "minimum C++14" that we do today).